docs/get-started/getting-started-alluxio.md

```diff
@@ -419,4 +419,145 @@ See the master.log and worker.log in this path.
+
+#### View a specific live Alluxio metrics in Prometheus Web UI
+The graph in the previous may not show all the metrics users care about,   
+the following describes how to view a specific historic Alluxio metric as you want:   
+Open Prometheus Web UI: http://localhost:39090/graph   
+Click the `Open metrics explorer` button.   
+![img](../img/Databricks/list-metrics.png)
+Then a list is shown:  
+![img](../img/Databricks/list-metrics-result.png)
+Select a metric and then click Graph Tab, then a graph is shown:   
+![img](../img/Databricks/a-metric-graph.png)
+
+### View Alluxio historic metrics based on Grafana with Prometheus
+This section is similar to [the previous section](#Monitor Alluxio live metrics based on Grafana with Prometheus)   
+After the Databricks cluster is shutdown, the Web UI on Databricks can not be accessed again,    
+this section describes how to view historic metrics.   
+The differences are:   
+- View historic metrics when Databricks cluster is shutdown   
+- Install and start Prometheus locally   
+- Restore the Prometheus data into local Prometheus   
+
+The steps are as following:   
+
+#### [Save Alluxio historic Prometheus-format metrics](#Save Alluxio historic Prometheus format metrics)
+
+#### [Install and start Grafana locally](#Install and start Grafana locally)
+
+#### Install and start Prometheus locally
+For example: local machine is Ubuntu.   
+```bash
+wget https://github.com/prometheus/prometheus/releases/download/v2.37.3/prometheus-2.37.3.linux-amd64.tar.gz   
+tar xvfz prometheus-*.tar.gz   
+cd prometheus-*   
+```
+For more details, refer to [doc](https://prometheus.io/docs/prometheus/latest/installation)   
+
+#### Restore historic metrics into Prometheus
+Copy the saved data in `PROMETHEUS_COPY_DATA_PATH` into Prometheus data path.   
+```
+cd <prometheus-root-dir>
+mkdir data   
+# Copy the saved files into `data` directory.
+cp -r $PROMETHEUS_COPY_DATA_PATH/some/sub/path/* /path/to/prometheus-root-dir/data
+`ls /path/to/prometheus-root-dir/data` will show files like:   
+`chunks_head  lock  queries.active  wal`
+```
+
+#### Start Prometheus
+cd <prometheus-root-path>   
+./prometheus   
+
+#### View Alluxio historic metrics based on Grafana
+Refer to [Config local Grafana to monitor the live metrics](#Config local Grafana to monitor the live metrics)   
+The difference is:   
+The prometheus datasource is local(http://localhost:9090) instead of the remote prometheus on Databricks cluster.   
+
+![img](../img/Databricks/grafana.png)
+
+#### View a specific historic Alluxio metrics in Prometheus Web UI
+Refer to [the section](#View a specific live Alluxio metrics in Prometheus Web UI)   
+The difference is:   
+The prometheus datasource is local instead of the remote prometheus on Databricks cluster.   
+Open Prometheus Web UI: http://localhost:9090/graph.   
+
+### Useful metrics for performance monitoring
+TODO: will add useful metrics   
```

Comment:
perhaps just remove this section for now and we can have follow PR

Reply:
Removed this section